testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -1585,6 +1587,584 @@ fn size_check_integration_test() {
     channel.stop_chains_coordinator();
 }
 
+#[test]
+#[ignore]
+fn size_overflow_unconfirmed_microblocks_integration_test() {
```

Comment:
Do you think these new tests should be added to the `bitcoin_tests` Github action? As long as they aren't flaky or add too much time to the action, we probably should.